chart/openfaas/templates/operator-rbac.yaml

```diff
@@ -64,5 +64,57 @@ subjects:
 - kind: ServiceAccount
   name: {{ .Release.Name }}-operator
   namespace: {{ .Release.Namespace | quote }}
+{{- if .Values.clusterRole}}
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ .Release.Name }}-operator-controller
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app: {{ template "openfaas.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: openaas-operator
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ .Release.Name }}-operator-controller
+  namespace: {{ .Release.Namespace | quote }}
+rules:
+  - apiGroups: ["openfaas.com"]
+    resources: ["functions"]
+    verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
```

Comment:
One comment/concern is that this uses a different format from the controller rbac, json arrays vs yaml arrays on new lines. 

This is a nitpick, but consistency makes it easier to maintain

Reply:
You are right Lucas, I was thinking how to approach this but it was consistent with the `Role` in `operator-rbac.yaml` that's why I went with that approach. Another thing is that in my opinion this is cleaner you see resource and verbs in two lines and you don't need to scroll. In general we can reformat this in another PR depending if we want json arrays or yaml arrays consistency.

Reply:
Consistency between the role for the operator and faas-netes would be a nice to have. @LucasRoesler if you would like to raise an issue to track it, we can prioritise that.